### PR TITLE
feat: set correct target when creating release

### PR DESCRIPTION
**Pull request by guilhermelimak**
Set the release branch name as target to avoid merging the changes commited to the main branch after the release PR is created to be included in the release.

References nearform/optic-release-automation-action#167

### CODE REVIEW

**Comment by guilhermelimak** [comment]
@simoneb This is ready for review

File: index.test.js [hunk 1]
```diff
@@ -58,3 +58,51 @@ test('create pr', async t => {
 
   t.same(response.statusCode, 200)
 })
+
+test('create release', async t => {
+  const mockedRepo = { repo: 'smn-repo', owner: 'salmanm' }
```
Comment (simoneb) [inline index.test.js:63]: is there an existing test which checks that this works if a target is NOT provided? I assume so. If not, make sure to create one. If it does, let's give this test a more meaningful name and maybe group together those two tests so we're colocating the two cases: one with and one without the target

Comment (guilhermelimak) [reply index.test.js:63]: Yep, this test was missing, just added it.

**Review by simoneb — APPROVED** [review]

**Comment by simoneb** [comment]
before we merge this, let's please give another thought about whether this is backwards compatible with the existing versions of the action circulating. I assume it is, but let's cross check as if that is not the case, everything will break

**Comment by guilhermelimak** [comment]
> before we merge this, let's please give another thought about whether this is backwards compatible with the existing versions of the action circulating. I assume it is, but let's cross check as if that is not the case, everything will break

I did test the backwards compatibility both ways but will do it again today just to make sure before deploying

**Comment by guilhermelimak** [comment]
> we can merge then, deployment is only done after a release

Just tested it again and all seems fine. I'll merge it

**Pull request merged**